test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -173,36 +209,60 @@ void test_ucp_ep_reconfig::entity::store_config()
     }
 
     m_cfg_index = ep()->cfg_index;
+
+    /* Calculate number of reused resources by:
+     * 1) Taking half of total lanes to be reused.
+     * 2) Dividing num_lanes by num_paths to get number of resources. */
+    auto num_reused   = (ucp_ep_num_lanes(ep()) / 2) / num_paths();

```

Comment:
try to remove num_paths() by the following logic: create a bitmap of all resources used by the endpoint, and remove half of its set bits